webapp/src/clj/lipas/backend/handler.clj

```diff
@@ -774,7 +773,42 @@
          :handler
          (fn [{:keys [body-params identity]}]
            {:status 200
-            :body   (core/save-ptv-integration-definitions db search identity body-params)})}}]]]
+            :body   (core/save-ptv-integration-definitions db search identity body-params)})}}]]
+
+     ;; legacy routes
+     ["/v1/api"
+      ["/swagger.json"
+       {:get
+        {:no-doc  true
+         :swagger {:id ::legacy
+                   :info {:title "Lipas-API (legacy) v1"}
+                   :securityDefinitions
+                   {:token-auth
+                    {:type "apiKey"
+                     :in   "header"
+                     :name "Authorization"}}}
```

Comment:
This can be left out, because we're removing the all the legacy endpoints that require Auth. AFAIK no one has ever used them. ;)

Reply:
Removed!